[tlparse] Add stacktrace filter utility

angelayi opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #146939
* #146955
* #146859
* __->__ #146858
* #146534
* #146533
* #146532

Added a utility function for capturing the user stack and framework stacktrace. 

cc @ezyang @SherlockNoMad @EikanWang @jgong5 @wenzhe-nrv

angelayi committed:
[tlparse] Add stacktrace filter utility

[ghstack-poisoned]

angelayi committed:
Update on "[tlparse] Add stacktrace filter utility"




cc ezyang SherlockNoMad EikanWang jgong5 wenzhe-nrv

[ghstack-poisoned]

bobrenjc93 reviewed (approved):
thanks for cleaning this up

angelayi committed:
Update on "[tlparse] Add stacktrace filter utility"


Added a utility function for capturing the user stack and framework stacktrace. 

cc ezyang SherlockNoMad EikanWang jgong5 wenzhe-nrv

[ghstack-poisoned]

angelayi committed:
Update on "[tlparse] Add stacktrace filter utility"


Added a utility function for capturing the user stack and framework stacktrace. 

cc ezyang SherlockNoMad EikanWang jgong5 wenzhe-nrv

[ghstack-poisoned]

angelayi committed:
Update on "[tlparse] Add stacktrace filter utility"


Added a utility function for capturing the user stack and framework stacktrace. 

cc ezyang SherlockNoMad EikanWang jgong5 wenzhe-nrv

[ghstack-poisoned]

angelayi committed:
Update on "[tlparse] Add stacktrace filter utility"


Added a utility function for capturing the user stack and framework stacktrace. 

cc ezyang SherlockNoMad EikanWang jgong5 wenzhe-nrv

[ghstack-poisoned]

angelayi committed:
Update on "[tlparse] Add stacktrace filter utility"


Added a utility function for capturing the user stack and framework stacktrace. 

cc ezyang SherlockNoMad EikanWang jgong5 wenzhe-nrv

[ghstack-poisoned]
